knative/client.go

```diff
@@ -8,6 +8,7 @@ import (
 	"time"
 
 	"k8s.io/client-go/tools/clientcmd"
+	clienteventingv1beta1 "knative.dev/client/pkg/eventing/v1beta1"
 	"knative.dev/client/pkg/kn/commands"
```

Comment:
would it be possible to avoid this dependency ? This won't work in the future when we carve out the client-pkg repo, there must be no dependency on any cobra command.